### PR TITLE
fix: Dashboard widgets do not align correctly

### LINKED ISSUE #1863: Dashboard widgets do not align correctly
## Bug description

The "Connect & transact" widgets do not align correctly with the "New in Safe" on large and medium resolutions.

## Steps to reproduce

Open the Safe on Gnosis Chain. Observe the "Connect & transact" widgets not stacking on larger resolutions and not spanning the height of the "New in Safe" on medium resolutions,

## Expected result

The "Connect & transact" widgets always stack, spanning the height of the "New in Safe" widget when it is present.

## Obtained result

The "Connect & transact" widgets do not always stack/span the height of the "New in Safe" widget.

## Screenshots

image

image

### PR DESCRIPTION
**Pull request by deepaksing**
## What it solves
This PR fixes the issue: fixes #1863 Dashboard widgets do not align correctly. Previously The "Connect & transact" widgets do not always stack/span the height of the "New in Safe" widget. This happened because **FeatureApps** Component was not able to take the full height of its parent component.
Resolves #

## How this PR fixes it
In this PR I gave 100% height to child component so that fixed the issue for medium sized screens. 
For large Sized screens the issue was with the min-width of the component. Due to this components were getting stacked horizontally. So increasing min-width to 500px for larger screen size solved the issue

## How to test it
Open the Safe on Gnosis Chain. Observe the "Connect & transact" widgets 
Connect & transact widget will span properly on larger resolutions and  span with the height of the "New in Safe" on medium resolutions,
## Screenshots

For Medium Sized Screens
image

For large Sized Screens
image

## Checklist
* [x] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/dashboard/FeaturedApps/FeaturedApps.tsx [hunk 1]
```diff
@@ -11,10 +11,14 @@ import SafeAppIconCard from '@/components/safe-apps/SafeAppIconCard'
...
 const StyledGrid = styled(Grid)`
   gap: 24px;
+  height:100%;
 `
 
 const StyledGridItem = styled(Grid)`
   min-width: 300px;
+  @media (min-width: 1460px) {
+    min-width: 500px;
+  }
```
Comment (usame-algan) [inline src/components/dashboard/FeaturedApps/FeaturedApps.tsx:21]: We can get rid of the min-width styles if we instead change the `flex-direction` on the `StyledGrid` element like so: `<StyledGrid container flexDirection={{ xs: 'column', sm: 'row', lg: 'column' }}>`

Comment (deepaksing) [reply src/components/dashboard/FeaturedApps/FeaturedApps.tsx]: HI, @usame-algan, i have made the required changes.

Comment (deepaksing) [reply src/components/dashboard/FeaturedApps/FeaturedApps.tsx]: @usame-algan  i messed up and now its showing unwanted changes in files changed. What should i do?

Comment (usame-algan) [reply src/components/dashboard/FeaturedApps/FeaturedApps.tsx]: Looks good! You could git reset back to the first commit on this Branch and push your changes again or revert the unwanted changes with another commit.

Comment (deepaksing) [reply src/components/dashboard/FeaturedApps/FeaturedApps.tsx]: @usame-algan, I have removed the unwanted changes.

**Review by usame-algan — APPROVED** [review]
Looks great, thanks for implementing the changes! 🚀

**Comment by katspaugh** [comment]
Thank you @deepaksing! 🙏

**Pull request merged**